mix.exs

```diff
@@ -9,13 +9,13 @@ defmodule Linguist.Mixfile do
 
   @repo_url "https://github.com/change/linguist"
 
-  @version "0.3.2"
+  @version "0.4.0"
 
   def project do
     [
       app: :linguist,
       version: @version,
-      elixir: "~> 1.6",
+      elixir: "~> 1.15",
```

Comment:
CI is still building on `1.10` which implies we still support back to that version, even with the `import` changes, so this should specify what is the min version we now support, and if we no longer support `1.6` should make that clear in the changelog

`ex_cldr:2.37.5` has `1.11`, so we should probably match that as the min we continue to support here in linguist

Reply:
`cldr_utils:2.24.2` also specifies `1.11` as the min version